process/archiving.md

```diff
@@ -0,0 +1,35 @@
+# CNCF Project Archiving Process v1.0
+
+Open source projects have a lifecycle and there are times that projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the TOC.
+
+## Archiving Criteria
+
+There are different criteria to consider when archiving a project, but here are ones that the TOC looks for:
```

Comment:
Would just one of these criteria make the project a candidate for archiving, or is the expectation that all of these things would likely be true?

Reply:
I feel this is a subjective call right? so i think one or more of the following should be true for TOC to look at archiving.